src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java

```diff
@@ -18,27 +20,38 @@
     private static final Supplier<Double> SEEK_ROTATE_VALUE = rootNamespace.addConstantDouble("seek rotate value", 0.5);
     private static final Supplier<Double> SEEK_HUB_TOLERANCE = rootNamespace.addConstantDouble("seek hub tolerance", 18);
 
+    private static final double AIM_TO_HUB_MOVE_VALUE = 0.55;
```

Comment:
this is quite slow and we don;t have the time

Reply:
This is temporary. I will change it when we calibrate.